src/support_sphere/lib/auth/login/login.dart

```diff
@@ -0,0 +1,147 @@
+import 'package:ionicons/ionicons.dart';
+import 'package:loading_overlay/loading_overlay.dart';
+import 'package:flutter/material.dart';
+import 'package:support_sphere/components/icon_logo.dart';
+import 'package:support_sphere/components/text_form_builder.dart';
+import 'package:support_sphere/components/password_form_builder.dart';
+import 'package:support_sphere/auth/register/register.dart';
+
+class Login extends StatefulWidget {
+  const Login({Key? key}) : super(key: key);
+
+  @override
+  _LoginState createState() => _LoginState();
+}
+
+class _LoginState extends State<Login> {
+  @override
+  Widget build(BuildContext context) {
+    // LoginViewModel viewModel = Provider.of<LoginViewModel>(context);
+    return LoadingOverlay(
+        isLoading: false,
+        child: Scaffold(
+          backgroundColor: Theme.of(context).colorScheme.onPrimary,
+          // key: viewModel.scaffoldKey,
+          body: ListView(
+              padding:
+                  const EdgeInsets.symmetric(horizontal: 15.0, vertical: 20.0),
+              children: [
+                SizedBox(height: MediaQuery.of(context).size.height / 5),
+                GestureDetector(
+                  onTap: () {
+                    Navigator.of(context).pop();
+                  },
+                  child: const IconLogo(),
+                ),
+                const SizedBox(height: 10.0),
+                Center(
+                  child: Text(
+                    'Log into your account and get started!',
```

Comment:
A person could be logging into their account after having gotten started. Maybe just `'Log into an existing account'`

-----

I wonder if there's a better way for these and other strings to be stored elsewhere and referenced here? Could be overkill, but if we could figure that out now, it's possible that we can make a prototype that could more easily support multiple languages in the future

Reply:
@nikiburggraf and @lsetiawan has chatted in person and will be storing strings to a `string_catalog` module in future changes.